services/api/app.py

```diff
@@ -34,10 +33,13 @@ def print_to_pdf(urls, title, subtitle):
 
 def render_book():
```

Comment:
This is just a question because I am stupid (ignorant, naive, etc) and had trouble following this function the first time that I saw it. 

Could we pass parameters to this function rather than relying on the global `request`? Ie, could we create some locals  from the `request` object in the route endpoint and pass those to `render_book`? I had a hard time grokking where `request` came from outside a function that has an obvious connection to an HTTP request. 

Thanks!!

Reply:
Great point!

This method existed before this PR so (if you don't mind!) I'd like to turn this observation into a new issue so it can be patched in a separate PR.

Reply:
Great point -- I was originally going to suggest we do this in another PR but realizing that I've upped the reliance on request in this PR to the point that it makes a lot of sense to just do it here and now.